### ISSUE TITLE
Specify hhvm.server.default_document more than once

### ISSUE THREAD

**Issue by Amitie10g** [facebook/hhvm#7817]
I'm running a tool in the Wikimedia Tool Labs and I proposed to add HHVM in server mode as one of the flavours of Kubernetes containers.

The implementation of HHVM in Proxygen mode seems fine, but I found an issue with the `hhvm.server.default_document` option: It can be set only once, and I'm unable to configure, for example, "index.html" in addition to "index.php", throwing a 404 error when accessing an URL without an index.php file but with an index.html one.

### HHVM Version
3.12.7

**Comment by mofarrell** [comment]
It currently only supports one default document type.  Is this blocking switching from fastcgi to proxygen?

**Comment by Amitie10g** [comment]
There are plans to implement HHVM in FastCGI mode with lighttpd, but this is not a priority task.

¿Is there plans to implement more than one default document in Proxygen mode?

**Comment by mofarrell** [comment]
We actually recommend running with proxygen over fastcgi (its faster).  However, I don't believe we have any plans to allow a set of default documents.  We would most likely accept a pull request for it though.

**Comment by lexidor** [comment]
Hi, sorry for the post from 2023. This feature has been supported using the virtual host syntax. This allows you to declare multiple default documents.
